### PR TITLE
add endpoint to update priority and status for admission

### LINKED ISSUE #1136: Interviewers set applicant status
Interviewers prioritizing applicants after interview. This is the "want", "reserve", "do not want" prioritization set by interviewers/admins during discussion at the end of the interview period.

Some of this has been started, so find out what has to be done before creating an issue from the tasks bellow. (hover over tasks to 'create issue' to the right)
- [x] #1157
- [x] #1158
- [ ] update applicant priority for other users
- [ ] implement backend api features in frontend

### PR DESCRIPTION
**Pull request by magsyg**
Added endpoint to update admission state for reccruiter status and priority. 
Own endpoint due to recruiter should edit interview, notes and state seperatly

Added 2 endpoints for position and gang, due to them having different permissions, and returning different data
closes #1136

### CODE REVIEW

**Review by Snorre98 — COMMENTED** [review]
@magsyg  merge master, se kommentarter

File: backend/samfundet/serializers.py [hunk 1]
```diff
@@ -756,6 +757,7 @@ class Meta:
         fields = '__all__'
 
     def update(self, instance: RecruitmentAdmission, validated_data: dict) -> RecruitmentAdmission:
+        # More or less this is rough, interview should be its own thing
```
[the remark this answers is not in the document]
Comment (magsyg) [reply backend/samfundet/serializers.py:760]: Er et for å separere Interview og Admission mer, slik at de postes seperat

[the remark this answers is not in the document]
Comment (Snorre98) [reply backend/samfundet/serializers.py:760]: Ok, men hvis noe må fikses så kan det ikke bare ligge som en kommentar i koden.

File: backend/samfundet/views.py [hunk 2]
```diff
@@ -741,6 +743,65 @@ def list(self, request: Request) -> Response:
         return Response(serializer.data)
 
 
+class RecruitmentAdmissionStateChoices(APIView):
```
[the remark this answers is not in the document]
Comment (magsyg) [reply backend/samfundet/views.py:796]: Done

File: backend/samfundet/serializers.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (magsyg) [reply backend/samfundet/serializers.py:824]: Done

**Review by Snorre98 — APPROVED** [review]

**Pull request merged**